quiche/src/packet.rs

```diff
@@ -1380,6 +1380,20 @@ mod tests {
         let hdr_num = u64::from(b.get_u24().unwrap());
         let pn = decode_pkt_num(0xace9fa, hdr_num, num_len);
         assert_eq!(pn, 0xace9fe);
+        // roundtrip
+        let base = 0xdeadbeef;
```

Comment:
Given the +1 in decode_pkt_num we should consider bases that end in 0xff in addition to other bases.